yt_dlp/extractor/generic.py

```diff
@@ -3478,6 +3593,55 @@ def filter_video(urls):
                 )
                 .*?
                 ['"]?file['"]?\s*:\s*["\'](.*?)["\']''', webpage))
+        if not found:
+            # Look for generic KVS player (must be before 'Broaden ...' sections)
+            found = re.search(r'<script [^>]*?src="https://.+?/kt_player\.js\?v=(?P<ver>(?P<maj_ver>\d+)(\.\d+)+)".*?>', webpage)
```

Comment:
> (must be before 'Broaden ...' sections)

?

Reply:
There is a block called `# Broaden the search a little bit`. The KVS code must be placed above it.